api/hypershift/v1beta1/zz_generated.featuregated-crd-manifests/hostedclusters.hypershift.openshift.io/AAA_ungated.yaml

```diff
@@ -169,6 +169,9 @@ spec:
                     items:
                       enum:
                       - ImageRegistry
+                      - Console
```

Comment:
Let's focus on one at a time. Let's pick e.g. console for this PR and defer any other changes to follow up PRs.